puppet/modules/dgu_ckan/manifests/init.pp

```diff
@@ -584,7 +584,7 @@
   # I can replicate this on the command-line when logged in as vagrant and using sudo,
   # but it works if you do 'sudo su' or 'sudo su co' and then do 'sudo npm install'.
   exec { 'npm_deps_dgu':
-    command   => 'sudo npm install',
+    command   => 'sudo npm install --no-bin-links',
```

Comment:
Is this a Windows thing? Are you running the whole VM off a network drive? Isn't the performance really bad?

I'm concerned that some of those symbolic links may be needed.